substrate/bin/node/cli/Cargo.toml

```diff
@@ -9,7 +9,6 @@ license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 default-run = "substrate-node"
 homepage = "https://substrate.io"
 repository.workspace = true
-publish = false
```

Comment:
This should not be published!

Reply:
They are deps of substrate so need to be published.

Reply:
@kianenigma can we not just provide a link to the `node` stuff? And to the `chain-spec-builder`?  (We can release the chain spec builder, when it doesn't depend on node-cli anymore)

Reply:
Yeah sorry, I added these things as deps to `substrate` just to use Rust's intra-crate linking convenience rather than building proper docs. You can remove the dependency, and fix the links. Although, it might be a bit iffy to fix them such that it works for both https://paritytech.github.io/ hosted docs and for docs.rs

For now feel free to not publish `substrate` as well. This is easier for us all and will solve your issue? 

- I will be moving `substrate` crate to a new `developer_hub` crate in an upcoming PR. 
- This new `developer_hub` cannot be published because of some other issues, so I am for now settling for the fact that `developer_hub` will always be `publish = false` and only accessible via https://paritytech.github.io/